tests/unit/components/legend.spec.ts

```diff
@@ -19,7 +19,7 @@ const appConfig = {
             unit: "%"
         }
     }
-};
+} as any;
```

Comment:
i dont personally think it is necesasary considering this is just a test, its actually a typescript problem in my opinion, it is complaining that we do not have `description`s in the indicators object but it would be nice if there was a way for  `Partial<>` type to be recursively applied since we dont need `description`s for the purposes of the test